crates/polars-core/src/chunked_array/logical/categorical/merge.rs

```diff
@@ -192,6 +199,33 @@ pub fn call_categorical_merge_operation<I: CategoricalMergeOperation>(
     }
 }
 
+struct DoNothing;
```

Comment:
I don't understand why we don't need to update the rhs. Can't there be duplicate entries?

Reply:
The physicals are already altered such that they are compatible so there is no need to do anything. 
Normally we do appends/extends/etc. here, that combine the two categoricals on the fly.

Reply:
Ah, right. I see the `call_categorical_merge_op` already fixes the cats. NVM. :+1: